plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui.test/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/flexible/FlexDeployCommandHandlerTest.java

```diff
@@ -100,12 +100,8 @@ public void testDeployContextMenu_hiddenForFlexJarNonMavenProject() {
   private static boolean flexDeployMenuVisible(IProject project) {
     SWTBotTreeItem selected = SwtBotProjectActions.selectProject(
         new SWTWorkbenchBot(), project.getName());
-    try {
-      selected.contextMenu("Deploy to App Engine Flexible...");
-      return true;
-    } catch (WidgetNotFoundException e) {
-      return false;
-    }
+    Predicate<String> isFlexDeployMenu = menu -> "Deploy to App Engine Flexible...".equals(menu);
+    return selected.contextMenu().menuItems().stream().anyMatch(isFlexDeployMenu);
```

Comment:
Is this really better than `.menuItems().contains("Deploy …")`?

Reply:
Ah, you're right.